frontend/src/styles/auth.css

```
@@ -0,0 +1,250 @@
+  height: 40px;
+  border-radius: 8px;
+  background: var(--auth-primary);
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 8px;
+}
+
+.auth-brand-name {
+  font-family: 'Manrope', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)